### DISCUSSION TITLE
[Marketplace] Libraries and cronjobs support

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#237] [Ideas]
### Which area of this project could be most improved?

Marketplace

### Description

The Mia-Platform marketplace now has microservices or group of microservices, or:
* templates
* plugins
* applications
* ...

What do you think about the addition of:
* libraries &rarr; there are many libraries on the public documentation, but are still missing in the marketplace. I think it would be useful to have in the marketplace the libraries and, by clicking on one of them, know how it can be installed and used inside the codebase;
* cronjobs &rarr; there are few cronjobs on the documentation (e.g. the Fast Data CSV Importer, if I am not wrong).

**Comment by davidebianchi** [comment]
Hi @silversoul93!
For the CronJob we already have planned it as a future improvement.
Great idea for the libraries, I think it could be really interesting and useful! @marcofilippi can we add it to the marketplace roadmap?

**Reply by frengooo** [reply]
@davidebianchi thanks for your reply.
I look forward to seeing the improvements on the CronJobs 😄

**Reply by marcofilippi** [reply]
@silversoul93 Thanks for the feedback, I keep track of this proposal inside of our roadmap!

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
